reference/data/models/pumpsettings/pumpsettings.v1.yaml

```diff
@@ -150,6 +150,44 @@ allOf:
         type: string
         minLength: 1
         maxLength: 100
+      sleepSchedules:
+        type: array
+        minItems: 0
+        maxItems: 10
```

Comment:
Is this imposed from some external spec, or just a _reasonable_ upper limit?

Reply:
It's just a reasonable upper limit.